src/appleseed/renderer/modeling/project/projectfilereader.cpp

```diff
@@ -3081,13 +3067,18 @@ auto_release_ptr<Project> ProjectFileReader::read(
             project_filepath, 
             unpacked_project_directory.c_str());
 
+        if (bf::exists(bf::path(unpacked_project_directory)))
+            bf::remove_all(bf::path(unpacked_project_directory));
+
         unzip(project_filepath, unpacked_project_directory);
         
+        const string project_name = appleseed_files[0];
         actual_project_filepath = (bf::path(unpacked_project_directory) / bf::path(project_name)).string().c_str();
         project_filepath = actual_project_filepath.data();
     }
 
     // Handle built-in projects.
```

Comment:
I would move back handling of built-in projects at the beginning of the function, like it used to be. Just getting this special case out of the way before checking for packet project files, etc.